### PR TITLE
OrbitControls: Add keydown event listener to domElement instead of window.

### PR DESCRIPTION
**Pull request by EliasHasle**
As requested in https://github.com/mrdoob/three.js/pull/16676#issuecomment-546588430 .

### CODE REVIEW

**Comment by Mugen87** [comment]
Can you please update the JSM version. In this way it's possible to make a few test with:

http://raw.githack.com/EliasHasle/three.js/orbitcontrols-domelement-keydown/examples/misc_controls_orbit.html

**Comment by EliasHasle** [comment]
Done.

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
Let's give this a try.

File: examples/js/controls/OrbitControls.js [hunk 1]
```diff
@@ -1136,7 +1136,15 @@ THREE.OrbitControls = function ( object, domElement ) {
 	scope.domElement.addEventListener( 'touchend', onTouchEnd, false );
 	scope.domElement.addEventListener( 'touchmove', onTouchMove, false );
 
-	window.addEventListener( 'keydown', onKeyDown, false );
+	scope.domElement.addEventListener( 'keydown', onKeyDown, false );
+
+	// make sure element can receive keys.
+
+	if ( scope.domElement.tabIndex === -1 ) {
```
Comment (WestLangley) [inline examples/js/controls/OrbitControls.js]: Formatting:

    if ( scope.domElement.tabIndex === - 1 ) {

But instead, I would avoid [guessing](https://github.com/mrdoob/three.js/pull/16676/files#r298252234) what the user's intention is and remove this test.

Comment (Mugen87) [reply examples/js/controls/OrbitControls.js]: Done #17826.

@EliasHasle Please consider to setup your IDE/editor in a way so this style issue are automatically detected.

Comment (WestLangley) [reply examples/js/controls/OrbitControls.js]: Guys. The white space is not the issue. Leave `tabIndex` to the application.

Comment (Mugen87) [reply examples/js/controls/OrbitControls.js]: Have you actually tested the example without this code?

Comment (WestLangley) [reply examples/js/controls/OrbitControls.js]: @Mugen87 I recall, now, that this came up before in #4327. See https://github.com/mrdoob/three.js/commit/ae9934776f383547ef136c13ed0b289a0d2abbb4.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**